File: js/961- N Repeated Element in Size 2N Array.js
var repeatedNTimes = function(A) {
  var total = 0, i, k; 
  for (i = 0; i < A.length; i++){ 
        for(k = i + 1; k < A.length; k++) {
            if(A[i]-A[k]==0){ 
                total = A[i]
            }
        }
  }  
    return total; 
};

File: leetcode/python/961- N-Repeated Element in Size 2N Array.py
""Using counter ""

import collections 
class Solution(object):
    def repeatedNTimes(self, A):
        n = len(A) /2
        a = collections.Counter(A)
        for i in range(len(a)): 
            if a.values()[i] == n: 
                return a.keys()[i]
                
 
